src/EnergyPlus/EconomicLifeCycleCost.cc

```diff
@@ -1507,11 +1496,9 @@ void ComputePresentValue(EnergyPlusData &state)
     // compute the Single Present Value factors based on the discount rate
     elcc->SPV.allocate(elcc->lengthStudyYears);
     for (int year = 1; year <= elcc->lengthStudyYears; ++year) {
-        std::map<DataGlobalConstants::ResourceType, Real64> yearMap;
-        for (ResourceType iResource : state.dataGlobalConst->AllResourceTypes) {
-            yearMap.insert(std::pair<DataGlobalConstants::ResourceType, Real64>(iResource, 0.0));
-        }
-        elcc->energySPV.insert(std::pair<int, std::map<DataGlobalConstants::ResourceType, Real64>>(year, yearMap));
+        std::array<Real64, static_cast<int>(Constant::ResourceType::Num)> yearMap;
+        std::fill(yearMap.begin(), yearMap.end(), 0.0);
+        elcc->energySPV[year] = yearMap;
```

Comment:
Here's another. Is yearMap used for anything else other than stuffing these economic arrays? If not then then could be a constexpr array?

Reply:
Same comment as above. Thanks.